code/datums/planet_loader/planet_loader.dm

```diff
@@ -37,6 +37,9 @@
 			PL.name_dock(D, D.encounter_type, params)
 			if(D.encounter_type == "main")
 				PL.main_dock = D
+			if(D.planet_dock)
+				var/obj/docking_port/stationary/fob_land/F = new(D.loc)
+				PL.docks |= F
```

Comment:
This code creates 2 docks

Reply:
Oops. Forgot about this when I was dealing with the two planet dock problem.